src/pathwayvis/components/reactionpanel/panelitem/panelitem.component.ts

```diff
@@ -13,11 +13,16 @@ class PanelItemCtrl {
   public onRemoveItem: (item: any) => void;
   public itemDisplay: (item: any) => string;
   public idProperty: string;
-
+  public objectiveReaction: string = null;
```

Comment:
What's the purpose of this? It's not added to the bindings. Also, I don't really like the name since it's too specific. If you look at `panelitem`, it's very generic, you could use it to build a horse race betting site! I'd either turn `items` => `item` and allow it to accept a single element and an array as well (and dispatch in the component). Or you could have `items` and `item` next to each other. if `items` is missing, `item` would be displayed.